turbopack/crates/turbopack-browser/src/chunking_context.rs

```diff
@@ -762,19 +764,19 @@ impl ChunkingContext for BrowserChunkingContext {
 pub async fn ident_to_output_filename(
     ident: Vc<AssetIdent>,
     context_path: Vc<FileSystemPath>,
-    expected_extension: RcStr,
+    _expected_extension: RcStr,
 ) -> Result<Vc<RcStr>> {
     let ident = &*ident.await?;
     let path = &*ident.path.await?;
-    let mut name = if let Some(inner) = context_path.await?.get_path_to(path) {
+    let name = if let Some(inner) = context_path.await?.get_path_to(path) {
         clean_separators(inner)
     } else {
         clean_separators(&ident.path.to_string().await?)
     };
-    let removed_extension = name.ends_with(&*expected_extension);
-    if removed_extension {
-        name.truncate(name.len() - expected_extension.len());
-    }
+    // let removed_extension = name.ends_with(&*expected_extension);
+    // if removed_extension {
+    //     name.truncate(name.len() - expected_extension.len());
+    // }
     // name += &expected_extension;
```

Comment:
i am not sure about this code here, it seems the code is useless here @xusd320 cc

Reply:
It's copied from next.js, I also not sure what's it for